.github/workflows/linux-ci-build.yml

```diff
@@ -0,0 +1,18 @@
+name: CI build
+
+on: [push]
+
+jobs:
+  Ubuntu:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Source
+      uses: actions/checkout@v1
+    
+    - name: Build and test
+      run: |
+        export CODEBUILD_SRC_DIR=$(pwd)
+        python3 -c "from urllib.request import urlretrieve; urlretrieve('https://raw.githubusercontent.com/awslabs/aws-c-common/builder2/codebuild/builder.py', 'builder.py')"
+        python3 builder.py build linux-clang-9-linux-x64
```

Comment:
confused about the difference between:
`linux-ci-action.yml` which does `python3 builder.py run aws-crt-python`
and
`linux-ci-build.yml` which does `python3 builder.py build linux-clang-9-linux-x64`

Why bother having `linux-ci-build.yml`?
Isn't the point of the builder2 experiment to stop using builder.json in repos like aws-crt-python where it only gets in the way?

Reply:
For comparison. Like I've been saying I would do :)